vapor-beta won't install

> % brew install vapor/tap/vapor-beta
==> Installing vapor-beta from vapor/tap
==> Downloading https://github.com/vapor/toolbox/archive/18.0.0-beta.19.tar.gz
==> Downloading from https://codeload.github.com/vapor/toolbox/tar.gz/18.0.0-beta.19
######################################################################## 100.0%
==> swift build --disable-sandbox
Last 15 lines from /Users/scott/Library/Logs/Homebrew/vapor-beta/01.swift:
/private/tmp/vapor-beta-20191219-36193-1g41ml/toolbox-18.0.0-beta.19/Sources/CloudCommands/Deploy.swift:9:10: error: cannot convert value of type 'Option<_>' to specified type 'String'
        @Option(name: "env", short: "e")
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/vapor-beta-20191219-36193-1g41ml/toolbox-18.0.0-beta.19/Sources/CloudCommands/Deploy.swift:11:10: error: cannot convert value of type 'Option<_>' to specified type 'String'
        @Option(name: "branch", short: "b")
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/vapor-beta-20191219-36193-1g41ml/toolbox-18.0.0-beta.19/Sources/CloudCommands/Deploy.swift:32:10: error: cannot convert value of type 'Option<_>' to specified type 'String'
        @Option(name: "app", short: "a")
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/vapor-beta-20191219-36193-1g41ml/toolbox-18.0.0-beta.19/Sources/CloudCommands/Deploy.swift:34:10: error: cannot convert value of type 'Option<_>' to specified type 'String'
        @Option(name: "env", short: "e")
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
/private/tmp/vapor-beta-20191219-36193-1g41ml/toolbox-18.0.0-beta.19/Sources/CloudCommands/Deploy.swift:36:10: error: cannot convert value of type 'Option<_>' to specified type 'String'
        @Option(name: "branch", short: "b")
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

--- a/Sources/CloudCommands/CloudGroup.swift
+++ b/Sources/CloudCommands/CloudGroup.swift
@@ -57,11 +57,11 @@ struct Logs: Command {
     // definition
     struct Signature: CommandSignature {
         @Option(name: "app", short: "a")
-        var app: String
+        var app: String?
         @Option(name: "env", short: "e")
-        var env: String
+        var env: String?
         @Option(name: "lines", short: "l")
-        var lines: Int
+        var lines: Int?
         @Flag(name: "show-timestamps", short: "t")
         var showTimestamps: Bool
     }

--- a/Sources/CloudCommands/Deploy.swift
+++ b/Sources/CloudCommands/Deploy.swift
@@ -5,11 +5,11 @@ import Globals
 struct CloudDeploy: Command {
     struct Signature: CommandSignature {
         @Option(name: "app", short: "a")
-        var app: String
+        var app: String?
         @Option(name: "env", short: "e")
-        var env: String
+        var env: String?
         @Option(name: "branch", short: "b")
-        var branch: String
+        var branch: String?
         @Flag(name: "force", short: "f")
         var force: Bool
         @Flag(name: "push", short: "p")
@@ -30,11 +30,11 @@ struct CloudDeploy: Command {
 struct CloudPush: Command {
     struct Signature: CommandSignature {
         @Option(name: "app", short: "a")
-        var app: String
+        var app: String?
         @Option(name: "env", short: "e")
-        var env: String
+        var env: String?
         @Option(name: "branch", short: "b")
-        var branch: String
+        var branch: String?
         @Flag(name: "force", short: "f")
         var force: Bool
     }
@@ -60,19 +60,19 @@ protocol BranchSignature {
 
 fileprivate struct _AppSignature: CommandSignature {
     @Option(name: "app", short: "a")
-    var app: String
+    var app: String?
 }
 
 
 fileprivate struct _EnvSignature: CommandSignature {
     @Option(name: "env", short: "e")
-    var env: String
+    var env: String?
 }
 
 
 fileprivate struct _BranchSignature: CommandSignature {
     @Option(name: "branch", short: "b")
-    var branch: String
+    var branch: String?
 }
 
 fileprivate struct _ForceSignature: CommandSignature {

--- a/Sources/CloudCommands/Git/RemoteGroup.swift
+++ b/Sources/CloudCommands/Git/RemoteGroup.swift
@@ -52,7 +52,7 @@ struct RemoteRemove: Command {
 struct RemoteSet: Command {
     struct Signature: CommandSignature {
         @Option(name: "app", short: "a", help: "the app to set")
-        var app: String
+        var app: String?
     }
 
     let help = "link your local repo to a cloud app."

--- a/Sources/CloudCommands/SSH/SSHAdd.swift
+++ b/Sources/CloudCommands/SSH/SSHAdd.swift
@@ -6,11 +6,11 @@ import CloudAPI
 struct SSHAdd: Command {
     struct Signature: CommandSignature {
         @Option(name: "readable-name", short: "n")
-        var readableName: String
+        var readableName: String?
         @Option(name: "path", short: "p")
-        var path: String
+        var path: String?
         @Option(name: "key", short: "k")
-        var key: String
+        var key: String?
     }
     
     let help = "add an ssh key to cloud."

--- a/Sources/CloudCommands/UserCommands/Login.swift
+++ b/Sources/CloudCommands/UserCommands/Login.swift
@@ -5,9 +5,9 @@ import Globals
 struct CloudLogin: Command {
     struct Signature: CommandSignature {
         @Option(name: "email", short: "e")
-        var email: String
+        var email: String?
         @Option(name: "password", short: "p")
-        var password: String
+        var password: String?
     }
 
     let help = "logs into vapor cloud."

--- a/Sources/CloudCommands/UserCommands/ResetPassword.swift
+++ b/Sources/CloudCommands/UserCommands/ResetPassword.swift
@@ -6,7 +6,7 @@ struct ResetPassword: Command {
 
     public struct Signature: CommandSignature {
         @Option(name: "email", short: "e")
-        var email: String
+        var email: String?
     }
 
     let help = "resets your account's password."

--- a/Sources/CloudCommands/UserCommands/Signup.swift
+++ b/Sources/CloudCommands/UserCommands/Signup.swift
@@ -5,15 +5,15 @@ import Globals
 struct CloudSignup: Command {
     struct Signature: CommandSignature {
         @Option(name: "first-name", short: "f", help: "your first name")
-        var first: String
+        var first: String?
         @Option(name: "last-name", short: "l", help: "your last name")
-        var last: String
+        var last: String?
         @Option(name: "org", short: "o", help: "name of your organization ('MyOrg')")
-        var org: String
+        var org: String?
         @Option(name: "email", short: "e", help: "email to signup with")
-        var email: String
+        var email: String?
         @Option(name: "password", short: "p", help: "the password to signupt with")
-        var password: String
+        var password: String?
     }
     
     let help = "creates a new account for vapor cloud."

--- a/Sources/VaporToolbox/New.swift
+++ b/Sources/VaporToolbox/New.swift
@@ -20,11 +20,11 @@ struct New: Command {
         
         // options
         @Option(name: "template", short: "t", help: templateHelp)
-        var template: String
+        var template: String?
         @Option(name: "tag", short: "t", help: "a specific tag to use, if desired.")
-        var tag: String
+        var tag: String?
         @Option(name: "branch", short: "b", help: "a specific branch to use, if desired.")
-        var branch: String
+        var branch: String?
     }
 
     let help = "creates a new vapor app from template. use 'vapor new project-name'."

